manual/src/cmds/runtime.etex

```diff
@@ -129,6 +129,8 @@ The following environment variables are also consulted:
         "caml_shutdown" in section~\ref{ss:c-embedded-code}). The option also enables
         pooling (as in "caml_startup_pooled"). This mode can be used to detect
         leaks with a third-party memory debugger.
+  \item[d] ("max_domains") Maximum number of domains that can be active
+        concurrently. Defaults to 128 in native and 16 in bytecode runtimes.
```

Comment:
Is that right? I thought it’s 128 for 64-bit arches and 16 for 32-bit arches? (i.e. bytecode on 64-bit is max 128?) but I’m only looking on my phone!

Reply:
David is right. Bytecode runtime on 64-bit platforms will default to 128.

Reply:
You are right. I read the definitions too fast.